### ISSUE TITLE
2.4.3 Focus Order (Contact Us)

### ISSUE THREAD

**Issue by AlanaBayless** [UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan#88]
**Were violation(s) found?**
(If Yes, describe in the comments.)
- [ ] No violations found
- [ ] Yes, violations found and all fixed
- [ ] Yes, violations found and some fixed while others are unresolved _(Add **6.Unresolved 508** label)_
- [ ] Yes, violations found but remain unresolved _(Add **6.Unresolved 508** label)_
- [ ] Flag as a potential problem _(Add **4.Help** label)_

---
### 2.4.3 Focus Order - Level A
If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability.

[Understanding 2.4.3](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)
[How to Meet 2.4.3](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=131%2C133%2C141%2C143%2C211%2C212%2C221%2C231%2C242%2C243#focus-order)
[How to Test 2.4.3](https://usfs.box.com/s/knn5w8jul8g0xicrnpa4cxzk07t8rc1d)

**Trusted Tester IDs:**
- [ ] Test 4.F focus-order meaning
- [ ] Test 4.G focus-order-reveal
- [ ] Test 4.H focus-order-return

**Accessibility Principle:** Operable (Navigable)

**Comment by AlanaBayless** [comment]
The Contact Us page fails 2.4.3 Focus Order, See which test ID's fail:
Trusted Tester IDs:

- [ ]  Test 4.F focus-order meaning: FAIL, When you tab through check box options the tab order is out of order. This is the current tab order:
1. Registered User
2. Administrator
**3. Forest Administrator (This is wrong)**
4. GIS Specialists
5. Regional Administrator

The correct tab order should follow:
1. Registered User
2. Forest Administrator
3. Administrator
4. GIS Specialists
5. Regional Administrator
![Image](https://github.com/UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan/assets/122119181/f59a06d9-19bd-45fb-92d2-eabc897faa0b)
- [ ]  Test 4.G focus-order-reveal: PASS
- [ ]  Test 4.H focus-order-return: PASS
